psi/pmtstreamtype.go

```diff
@@ -40,6 +40,8 @@ const (
 	PmtStreamTypeScte35 uint8 = 134 // SCTE-35
 
 	PmtStreamTypeID3 uint8 = 21 // Nielsen ID3
+
+	PmtStreamTypePrivateContent uint8 = 6 // Subtitle
```

Comment:
This is used to signal EAC-3 in DVB land... Which is pretty obnoxious. You may want to just leave the comment indicating `private data` rather than `Subtitle`.

Reply:
Agreed. Changed to Private Content.